<ui version="4.0" >
 <class>AddDialog</class>
 <widget class="QWidget" name="AddDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add/Import</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="1" >
    <widget class="QPushButton" name="btnCancel" >
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="btnAdd" >
     <property name="text" >
      <string>Add Program</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Import</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="2" >
       <widget class="KUrlRequester" name="kurlfrScript" />
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="btnImport" >
        <property name="text" >
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="chkSymlink" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Instead of copying a link will be made to the script or program, this is useful for keeping a copy easily accessable in your home directory instead of having to go into your ~/.kde/shutdown to make modifications.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="whatsThis" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Instead of copying a link will be made to the script or program, this is useful for keeping a copy easily accessable in your home directory instead of having to go into your ~/.kde/shutdown to make modifications.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Symlink</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kurlfrScript</tabstop>
  <tabstop>chkSymlink</tabstop>
  <tabstop>btnImport</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
